tensorflow/contrib/tensorrt/convert/convert_nodes.h

```diff
@@ -38,7 +38,7 @@ tensorflow::Status ConvertSubGraphToTensorRTNodeDef(
         input_inds,  // {node_id, output_idx}
     const std::vector<std::pair<int, int>>&
         output_inds,  // {node_id, output_idx}
-    size_t max_batch_size, size_t max_workspace_size_bytes_bytes,
+    size_t max_batch_size_bytes, size_t max_workspace_size_bytes,
```

Comment:
Revert the change to max_batch_size? I think it has no units.

Reply:
Oops :)